res/matrix-org/invite_template.eml

```diff
@@ -16,8 +16,9 @@ Hi,
 %(sender_display_name)s %(bracketed_verified_sender)shas invited you into a room
 %(bracketed_room_name)son Matrix. To join the conversation, either pick a
 Matrix client from https://matrix.org/docs/projects/try-matrix-now.html or use
-the single-click link below to join via Element (requires Chrome, Firefox,
-Safari, iOS or Android)
+the single-click link below to join via Element requires Chrome, Firefox,
+Safari, iOS or Android
+
 
```

Comment:
also needs reverting.

you can do `git checkout main -- invite_template.eml` in the same directory to re-check-out the email template from the main branch (thus reverting your changes to it).